src/main/java/com/google/api/codegen/go/GoDiscoveryContext.java

```diff
@@ -147,4 +148,28 @@ public String getApiVersion() {
     String rename = API_VERSION_RENAME.get(getApi().getName(), getApi().getVersion());
     return rename == null ? getApi().getVersion() : rename;
   }
+
+  /*
+   * Returns an empty or singleton list of auth scope for the method. If the method has no scope,
+   * returns an empty list, otherwise returns the first scope. We return an empty list instead of
+   * null to denote absence of scope since snippet cannot handle null values. The scope, if exists,
```

Comment:
s/if exists/if it exists/

or better yet:

"If the scope exists, it is stripped...."